backend/gamecreation/views.py

```
@@ -6,35 +6,243 @@
+                    params={"jql": importValue, "maxResults": 100}  
+                )
+    
+                data=response.json()
+                print(data)
+                issues_list = data['issues']
+                for jira_data in issues_list :
+                    ticket_key=jira_data["key"]
+                    summary = jira_data["fields"]["summary"]
+                    priority=jira_data["fields"]["priority"]["name"]
+                    description = jira_data["fields"]["description"]["content"][0]["content"][0]["text"]
+                    issue_type = jira_data["fields"]["issuetype"]["name"] 
```

Ruff (0.12.2)

Local variable `issue_type` is assigned to but never used

Remove assignment to unused variable `issue_type`

(F841)